config/v1/types_image.go

```diff
@@ -37,6 +37,23 @@ type Image struct {
 	Status ImageStatus `json:"status"`
 }
 
+// ImportModeType describes how to import an image manifest.
+// +enum
```

Comment:
What's this tag for? I don't recognise it

Reply:
which ? the enum?

Reply:
Yep, I'd have expected something like `// +kubeuilder:validation:Enum:=A;B;C

Reply:
remove this enum tag removes the enum listing from the [openapi](https://github.com/openshift/api/pull/1928/files#diff-6466e6ec9d299ee55a89f9ae0f455bc960d7295ead963490f5497447c2f1feb3R7224) files. is that ok ?

Reply:
Hmm, lets keep it. Might need to look at this tag again, I'll add a note to self